src/main/java/org/generationcp/middleware/api/breedingmethod/BreedingMethodDTO.java

```diff
@@ -1,15 +1,23 @@
 package org.generationcp.middleware.api.breedingmethod;
 
+import org.generationcp.middleware.api.program.ProgramFavoriteDTO;
 import org.generationcp.middleware.pojos.Method;
 import org.pojomatic.Pojomatic;
 import org.pojomatic.annotations.AutoProperty;
 
+import java.util.Date;
+import java.util.List;
+
 @AutoProperty
 public class BreedingMethodDTO extends BreedingMethodNewRequest {
 
 	private Integer mid;
 	private Boolean isBulkingMethod;
 	private Boolean isFavorite;
+	private Date date;
```

Comment:
@cuenyad should we add JsonFormat to this? 
I think we should rename it to something more representative like creationDate